### PR TITLE
Stage/cci

### PR DESCRIPTION
**Pull request by tejohnso**
@rodrigopavezi @stulees this is now building and working in launcher. There may still be some glitches which I'll address during the e2e work. The build packages the source and node_modules into an asar file which is then wrapped in self executing zips as launcher was.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+MODULENAME="player"
+VERSION=$(cat version)
+echo "deploying $VERSION"
+
+gsutil -m cp -p gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/* 
+mkdir -p manifests
+gsutil cp gs://install-versions.risevision.com/display-modules-*.json manifests
+
+find manifests -name "*.json" -exec node update-module-version.js '{}' $MODULENAME $VERSION 0
```
Comment (stulees) [inline deploy.sh]: @tejohnso Can you clarify what results in executing this line and in conjunction with what `update-module-version.js` does? Cheers

Comment (tejohnso) [reply deploy.sh]: @stulees update-module-version updates the remote manifest.  

We have something like:

``` js
{
  "rolloutPct": 0,
  "modules": [
    {
      "name": "launcher",
      "version": "13245"
    },
    {
      "name": "player",
      "version": "13245"
    }
  ]
}
```

And we need to potentially update the rolloutPct and / or update the version for one of the modules. We were going to use *jq* but the command for that turned out to be pretty hairy so I create a node script instead.  It runs as

``` bash
node update-module-version.js [path-to-local-copy-of-manifest] [module-to-change] [version-to-set] [new-rollout-pct]
```
So the result of the line you're asking about is that each of the manifests (one per arch) gets its player module version updated to the current rollout version.

Comment (stulees) [reply deploy.sh]: @tejohnso Nice, thanks for clarifying. So what is the scenario where you'd have an actual path to apply to `[path-to-local-copy-of-manifest]` when executing `node update-module-version.js`?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+gsutil cp manifests/*.json gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION
+gsutil setmeta -h "Cache-Control:private, max-age=0" gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/*
+gsutil setmeta -h "Content-Disposition:attachment" gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/*.sh
+gsutil acl ch -u AllUsers:R gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/*
+gsutil cp -p gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/* gs://install-versions.risevision.com/backups/$MODULENAME/$VERSION
+gsutil -m cp -p gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/* gs://install-versions.risevision.com/
+gsutil -m cp -p gs://install-versions.risevision.com/staging/$MODULENAME/$VERSION/* gs://install-versions.risevision.com/releases/$MODULENAME/$VERSION
+
+echo -n "$VERSION" > latest-version
+gsutil cp latest-version gs://install-versions.risevision.com
```
Comment (stulees) [inline deploy.sh]: @tejohnso Are we designating player-electron as the module to refer to `latest-version` for Support? I don't know how relevant this is going to stay when we break this module out further and along with other modules.

**Pull request merged**